cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,81 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import ArticlePage from '../support/pages/article.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import faker from 'faker';
+import '../support/commands';
+
+const signInPage = new SignInPageObject();
+const homePage = new HomePageObject();
+const articlePage = new ArticlePage();
 
 describe('Article', () => {
-  before(() => {
+  let user;
 
+  let title;
+  let description;
+  let content;
+  let tags;
+
+  before(() => {
+    cy.task('db:clear');
```

Comment:
The 'db:clear' task is run before each test suite but not before each test. This could lead to tests affecting each other and causing false positives/negatives.

Reply:
I made it because Marina Bidnenko had commented earlier:
It's not recommended to clear the database before each test. It might cause performance issues. Consider resetting the state in the database only when necessary.
https://github.com/mate-academy/qa_cypress_e2e_project/pull/59#discussion_r1285019476